### PR TITLE
implementing @forbig_web_access decorator

### PR DESCRIPTION
**Pull request by JankesJanco**
Resolves corresponding issue #102

### CODE REVIEW

**Comment by babky** [comment]
Unit tests are missing and failing :(.

File: src/zsl/task/task_decorator.py [hunk 1]
```diff
@@ -485,3 +485,15 @@ def crossdomain_inner_fn(*args, **kwargs):
...
 
     return decorator
+
+
+def forbid_web_access(f):
+
+    @wraps(f)
+    def wrapper_fn(*args, **kwargs):
+        if isinstance(JobContext.get_current_context(), WebJobContext):
+            raise Exception('Access forbidden from web.')
```
[the remark this answers is not in the document]
Comment (babky) [reply src/zsl/task/task_decorator.py]: This still applies.

File: src/zsl/task/task_decorator.py [hunk 2]
```diff
@@ -485,3 +485,15 @@ def crossdomain_inner_fn(*args, **kwargs):
         return crossdomain_inner_fn
 
     return decorator
+
+
+def forbid_web_access(f):
+
```
[the remark this answers is not in the document]
Comment (babky) [reply src/zsl/task/task_decorator.py]: and also add python documentary comments please

[the remark this answers is not in the document]
Comment (JankesJanco) [reply src/zsl/task/task_decorator.py]: done

File: src/zsl/task/task_decorator.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (JankesJanco) [reply src/zsl/task/task_decorator.py]: I've created new `ForbiddenError` (located in `zsl/interface/task` - let me know if it is not good location) and corresponding error handler.
I also created test case, see `tests.interface.web.http_forbidden_task_test`. I also wanted to create test which would execute "http forbidden" task in gearman or celery worker context but I don't know how.

[the remark this answers is not in the document]
Comment (babky) [reply src/zsl/task/task_decorator.py]: I would rather place that exception into `zsl/interface/web`

[the remark this answers is not in the document]
Comment (babky) [reply src/zsl/task/task_decorator.py]: Otherwise ok.

**Pull request merged**